prime-router/src/main/kotlin/azure/ApiKeysFunctions.kt

```diff
@@ -85,15 +84,18 @@ class ApiKeysFunctions(private val settingsFacade: SettingsFacade = SettingsFaca
 
             val updatedKeys = JwkSet.addKeyToScope(organization.keys, scope, jwk)
 
-            settingsFacade.putSetting(
-                SettingType.ORGANIZATION.name,
+            val (result, error) = settingsFacade.putSetting(
+                organization.name,
```

Comment:
This was a typo from doing a refactor before merging, but was not caught do to how the mocking was setup in the tests